.github/workflows/documentation-build.yaml

```diff
@@ -22,3 +25,8 @@ jobs:
       - name: Generate Site
         working-directory: ./docs
         run: npm run build
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          path: docs/build/site
+          retention-days: 1
```

Comment:
Why do we need this step? Debugging purposes?

Reply:
Yes